packages/react-components/react-portal/src/components/Portal/usePortalMountNode.ts

```diff
@@ -48,26 +44,42 @@ export const usePortalMountNode = (options: UsePortalMountNodeOptions): HTMLElem
     return [newElement, () => newElement.remove()];
   }, [targetDocument]);
 
-  // This useEffect call is intentional
-  // We don't want to re-create the portal element when its attributes change.
-  // This also should not be done in an effect because, changing the value of css variables
-  // after initial mount can trigger interesting CSS side effects like transitions.
-  useInsertionEffect(() => {
-    if (!element) {
-      return;
-    }
+  if (reactMajorVersion >= 18) {
+    // @ts-expect-error useInsertionEffect does not exist on React 17 types
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    React.useInsertionEffect(() => {
```

Comment:
I can't add a suggestion, but let's use `(React as never)['useInsertion' + 'Effect']` as it was originally, otherwise it cause problems with bundlers

Reply:
Fixed in https://github.com/microsoft/fluentui/pull/26377/commits/67ee7d02f8ecb27065adb90863489326e5996e40